src/common/secrets/index.ts

```diff
@@ -37,6 +37,7 @@ export const SecretKeyList = {
   ELASTICSEARCH_NODE: process.env.ELASTICSEARCH_NODE || null,
   ELASTICSEARCH_USERNAME: process.env.ELASTICSEARCH_USERNAME || null,
   ELASTICSEARCH_PASSWORD: process.env.ELASTICSEARCH_PASSWORD || null,
+  VUE_APP_PINATA_JWT_KEY: process.env.VUE_APP_PINATA_JWT_KEY || null,
```

Comment:
change to `PINATA_JWT` and please add to https://github.com/debionetwork/debio-backend/blob/main/.env-template

Reply:
done